lib/db/dao/activity_template_dao.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:async';
+import 'package:maui/app_database.dart';
+import 'package:sqflite/sqflite.dart';
+import '../entity/activity_template.dart';
+
+class ActivityTemplateDao {
+  Future<ActivityTemplate> getTopic(String id, {Database db}) async {
```

Comment:
Since there is no primary key for the ActivityTemplate table, it is best to not have this method since we cannot guarantee one unique row